pkg/apis/awsproviderconfig/v1beta1/awsmachineproviderconfig_types.go

```diff
@@ -25,9 +25,9 @@ import (
 const (
 	// ClusterIDLabel is the label that a machineset must have to identify the
 	// cluster to which it belongs.
-	ClusterIDLabel   = "sigs.k8s.io/cluster-api-cluster"
-	MachineRoleLabel = "sigs.k8s.io/cluster-api-machine-role"
-	MachineTypeLabel = "sigs.k8s.io/cluster-api-machine-type"
+	ClusterIDLabel   = "machine.openshift.io/cluster-api-cluster"
```

Comment:
@spangenberg this will never pass the test and will break the installer unless you make it backward compatible since it's used by `getClusterID`